### DISCUSSION TITLE
Static file service conflicts with dynamic routing

### DISCUSSION THREAD

**Discussion by jiangqm** [vercel/next.js#47726] [Help]
### Summary

I have set a dynamic route /[urlHandle] on the root route of the page. At this time, the favicon.ico in the/public file cannot be accessed through the path/public/favicon.ico after packaging. The official document indicates that there is an error, but the actual business scenario has this situation. How can I solve this problem?

**Comment by GustavoMelloGit** [comment]
Similar problem here
